### PR TITLE
[OSDOCS-5364]: 4.14z: Removing TP wording for hosted control planes

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OSDOCS-5364

Link to docs preview:
- Hosted control planes overview: https://67825--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/
- Configuring hosted control planes: https://67825--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-configuring
- Backup, restore, and disaster recovery for hosted control planes: https://67825--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-backup-restore-dr#hcp-backup-restore
- Troubleshooting hosted control planes: https://67825--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-troubleshooting

QE review:
- [x] QE has approved this change.

Additional information:
This PR is for the ACM 2.9/MCE 2.4 release that is scheduled for Nov. 20. Do not merge until this date.

### CODE REVIEW

**Comment by zanetworker** [comment]
Should we keep the red disclaimer for AWS and Power/Z?

**Comment by lahinson** [comment]
> Should we keep the red disclaimer for AWS and Power/Z?

@zanetworker Good question. Typically, we add that red boilerplate statement at the module level rather than within a subsection. I can add the boilerplate to the AWS and IBM Z sections and see whether the OCP doc peer review team feels strongly about it.

**Comment by zanetworker** [comment]
Thanks Laura. 

The other comment is as discussed. Aligning how we guide for `dump` commands between Hypershift and MCE (where the dump for must-gather is currently integrated).  https://67825--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-troubleshooting

**Comment by lahinson** [comment]
@zanetworker Thanks for your feedback! I moved the changes from PR #66571 to this one, so the must-gather updates on the Troubleshooting page will be delivered through this PR with the TP wording changes.

File: hosted_control_planes/hcp-configuring.adoc [hunk 1]
```diff
@@ -8,45 +8,46 @@ toc::[]
...
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-bm[Configuring the hosting cluster on bare metal]:
 ** Configure DNS
 ** Create an `InfraEnv` resource and add agents to it
 ** Create a hosted cluster and verify cluster creation
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal]:
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal]:
 ** Scale the `NodePool` object for a hosted cluster
-** Handle Ingress for a hosted cluster
+** Handle ingress for a hosted cluster
```
Comment (skrthomas) [inline hosted_control_planes/hcp-configuring.adoc:23]:
```suggestion
** Handle ingress traffic for a hosted cluster
```

Comment (skrthomas) [reply hosted_control_planes/hcp-configuring.adoc:23]: I would suggest adding "traffic" here

**Review by skrthomas — COMMENTED** [review]
Only one small comment, lgtm otherwise :)

File: hosted_control_planes/hcp-configuring.adoc (hunk 1 above)
Comment (lahinson) [reply hosted_control_planes/hcp-configuring.adoc:23]: Good suggestions -- thanks @skrthomas!

**Comment by lahinson** [comment]
/label merge-review-needed

Note to reviewers: This PR should not be merged until Nov. 20, in alignment with the ACM 2.9 GA. I'm requesting a merge review in advance per the new process: https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.x8ydklaezdcq

File: modules/hosted-control-planes-overview.adoc [hunk 2]
```diff
@@ -6,14 +6,11 @@
...
 
 You can use hosted control planes for Red Hat {product-title} to reduce management costs, optimize cluster deployment time, and separate management and workload concerns so that you can focus on your applications.
 
-You can enable hosted control planes as a Technology Preview feature by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#cluster_mce_overview[multicluster engine for Kubernetes operator version 2.0 or later] on Amazon Web Services (AWS), bare metal by using the Agent provider, or {VirtProductName}.
-
-:FeatureName: Hosted control planes
-include::snippets/technology-preview.adoc[]
+You can enable hosted control planes by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#cluster_mce_overview[multicluster engine for Kubernetes operator version 2.0 or later] on bare metal by using the Agent provider, or {VirtProductName}. Hosted control planes is available as a Technology Preview feature on the Amazon Web Services and {ibm-z-title} platforms.
```
Comment (maxwelldb) [inline modules/hosted-control-planes-overview.adoc:13]: "Amazon Web Services" -> "Amazon Web Services (AWS)" or attribute?

**Review by maxwelldb — APPROVED** [review]
lgtm. 2.9 links should be clicked through prior to merge.

File: modules/hosted-control-planes-overview.adoc (hunk 2 above)
Comment (lahinson) [reply modules/hosted-control-planes-overview.adoc:13]: Good catch. I didn't think to check for an attribute. It looks like there is one. I'll update that. Thanks!

**Pull request merged**